doc-content/enterprise-only/getting-started-kogito/proc-kogito-migrate-drl-expose-ruleunits.adoc

```diff
@@ -0,0 +1,131 @@
+[id="proc-kogito-migrate-drl-expose-ruleunits_{context}"]
+= Exposing rule units with a REST endpoint using {QUARKUS}
```

Comment:
There isn't any rule unit up to this point, they will be introduced only with the last step. Please rephrase this as

`Exposing rules evaluation with a REST endpoint using {Quarkus}`

Reply:
Done